tests/Integration/Behaviour/Features/Context/Domain/Product/Combination/UpdateCombinationOptionsFeatureContext.php

```diff
@@ -59,14 +58,14 @@ public function updateOptions(string $combinationReference, TableNode $tableNode
      */
     public function assertOptions(string $combinationReference, CombinationOptions $expectedOptions): void
     {
-        $combinationForEditing = $this->getCombinationForEditing($combinationReference);
         $optionPropertyNames = ['ean13', 'isbn', 'mpn', 'reference', 'upc'];
+        $actualOptions = $this->getCombinationForEditing($combinationReference)->getOptions();
         $propertyAccessor = PropertyAccess::createPropertyAccessor();
 
         foreach ($optionPropertyNames as $propertyName) {
             Assert::assertSame(
                 $propertyAccessor->getValue($expectedOptions, $propertyName),
-                $this->extractValueFromCombinationForEditing($combinationForEditing, $propertyName),
```

Comment:
@jolelievre actually this wasn't worth doing. It doesn't have as much use as in Product entity.